CONTRIBUTING.md

```diff
@@ -1,195 +1,137 @@
-   * Use comments on the issue itself to indicate that you will be working on it and get guidance and help.
-
-## Communicating with the team
-
-[Ballerina-Dev Google Group](https://groups.google.com/forum/#!forum/ballerina-dev) is the main Ballerina project discussion forum.
-
-[StackOverflow](https://stackoverflow.com/questions/tagged/ballerina) is used for support, [Slack](https://ballerinalang.slack.com/) for real-time communications, and [GitHub](https://github.com/ballerina-platform/ballerina-lang/issues) for issues and code repositories.
-
-## Filing issues
-
-If you are unsure whether you have found a bug, please consider searching existing issues in github and asking in Ballerina-Dev Google Group.
-
-> IMPORTANT: Sensitive security-related issues should be reported to [security@ballerina.io](security@ballerina.io). For details, see the [security policy](https://ballerina.io/security/).
-
-To file non-security issues:
-
-1. Click the **Issues** tab in the github repository,
-
-2. Click the **New Issue** button,
+- [Get started](#get-started)
+- [Set up the Prerequisites](#set-up-the-prerequisites)
+- [Obtain the source code](#obtain-the-source-code)
+- [Build the project](#build-the-project)
+- [Set up the development environment](#set-up-the-development-environment)
+    - [Set up the Ballerina plugin](#set-up-the-ballerina-plugin)
+    - [Set up the ANTLR4 plugin](#set-up-the-antlr4-plugin)
+- [Contribute to Ballerina grammar](#contribute-to-ballerina-grammar)
+- [Submit your contribution](#submit-your-contribution)
+- [Accept the Contributor License Agreement](#accept-the-contributor-license-agreement)
+- [Propose changes to Ballerina](#propose-changes-to-ballerina)
 
-3. Fill out all sections in the issue template and submit.
+## Get started
 
-## Contributing site changes, docs, and examples
+- Join the conversations at:
 
-## Contributing code
+ <div class="col-sm-12 col-md-12 cBallerinaLearnWrapper cGrayWrapper cCommunityLinks">
+  <div class="col-sm-12 col-md-3 cBoxContainer cCommunityIconsContainer">
+      <div class="cBallerina-Box">
+      <a class="cCommunityIcons" href="https://twitter.com/ballerinalang"><img src="/img/twitter.svg"></a>
+      </div>
+   </div>
+   <div class="col-sm-12 col-md-3 cBoxContainer cCommunityIconsContainer">
+      <div class="cBallerina-Box">
+      <a class="cCommunityIcons" href="/community/slack"><img src="/img/slack.svg"></a>
+      </div>
+   </div>
+      <div class="col-sm-12 col-md-3 cBoxContainer cCommunityIconsContainer">
+      <div class="cBallerina-Box">
+      <a class="cCommunityIcons" href="https://stackoverflow.com/questions/tagged/ballerina"><img src="/img/stackoverflow.svg"></a>
+      </div>
+   </div>
+   <div class="col-sm-12 col-md-3 cBoxContainer cCommunityIconsContainer">
+      <div class="cBallerina-Box">
+      <a class="cCommunityIcons" href="https://groups.google.com/forum/#!forum/ballerina-dev"><img src="/img/google-groups.svg"></a>
+      </div>
+   </div>
+</div>
```

Comment:
How the above stuff including images working in the .md file? Seems we need to reformat this to text along with relevant links.

Reply:
Added links instead of the HTML icons.